tests/libvmi/factory.go

```diff
@@ -32,7 +32,7 @@ import (
 // This image has tooling for the guest agent, stress, SR-IOV and more.
 func NewFedora(opts ...Option) *kvirtv1.VirtualMachineInstance {
 	fedoraOptions := []Option{
-		WithResourceMemory("512M"),
+		WithResourceMemory("1Gi"),
```

Comment:
Why?

Reply:
That was temporary to see if it would fix my tests, which it did!
Fedora + EFI won't boot with only 512M of memory.
I'll write a specific helper for that use-case and revert this.